package-lock.json

```diff
@@ -20,10 +20,10 @@
       "dev": true,
       "requires": {
         "@babel/types": "7.0.0-beta.51",
-        "jsesc": "^2.5.1",
```

Comment:
I think at least some of these changes are a result of your npm version being different from mine.  Probably don't merge them.